api-docs/list-leads.md

```diff
@@ -0,0 +1,49 @@
+# View a list of leads
```

Comment:
Can't this documentation just be merged into https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/leads.md#get-all-leads instead of having its own file?